docs/images.asciidoc

```diff
@@ -0,0 +1,42 @@
+[id="{p}-custom-images"]
+== Creating custom images
+
+It is possible to create your own custom Elasticsearch (or Kibana, etc.) image instead of using the base image provided by Elastic. You might want to do this to preload plugins in the image rather than having to install them via init container each time a pod starts. For example, if you want to create an Elasticsearch 7.3.2 image with the https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-gcs.html[Google Cloud Storage Repository Plugin], you could create a `Dockerfile` containing
+
+----
+FROM docker.elastic.co/elasticsearch/elasticsearch:7.3.2
```

Comment:
Should we mention that you have to use the official image as the base image for things to work out?